spec/fixtures/encrypted.json

```diff
@@ -1,23 +1,24 @@
 {
-  "not_encrypted": "plain text value",
   ".encrypted": {
-    ".key": "HJOZVMkbWggNQjOZRc8b+A==\n|e835e0af",
-    "foo": "yL3tigR7MUX3aQ3maxvKJw==\n|a629e5ad",
+    ".salt": "aecdfdb296983ec0",
+    ".key": "$AES$:LNkaWu/g7gM7zu4qC/4FAGOANOLWcY86uqxQfFiHRVSvXSA23pY=\n",
```

Comment:
How about we chomp the base64 values to get rid of the annoying trailing new lines. Bonus points for scrubbing the `=` as well. We only want to strip the trailing ones since the line feeds in the middle make the values much easier for text editors to deal with when using YAML.